ocrd_validators/ocrd_validators/workspace_validator.py

```diff
@@ -141,13 +143,27 @@ def _validate_dimension(self):
             if page.imageWidth != exif.width:
                 self.report.add_error("PAGE '%s': @imageWidth != image's actual width (%s != %s)" % (f.ID, page.imageWidth, exif.width))
 
+    def _validate_multipage(self):
+        """
+        Validate the number of images per file is 1 (TIFF allows multi-page images)
+
+        See `spec <https://ocr-d.github.io/mets#no-multi-page-images>`_.
+        """
+        for f in [f for f in self.mets.find_files() if f.mimetype.startswith('image/') and f.url]:
```

Comment:
`f.url` test is only used for two validation methods. Intended?

Reply:
IIRC that was to guard against `<mets:file>` w/o FLocat. Which is actually allowed in METS, annoyingly. Better not to be lenient, I'll remove the check and see what breaks :)

Reply:
Or even better, make that a separate check in the validator.

Reply:
Is now checked in `_validate_mets_files`